pkg/aws/plugin_test.go

```diff
@@ -0,0 +1,80 @@
+//go:build unit
+
+/*
+Copyright 2024 The Paraglider Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws
+
+import (
+	"testing"
+
+	orchestrator "github.com/paraglider-project/paraglider/pkg/fake/orchestrator/rpc"
+	"github.com/paraglider-project/paraglider/pkg/paragliderpb"
+	"github.com/paraglider-project/paraglider/pkg/utils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateResource(t *testing.T) {
```

Comment:
Should there be tests for the availability zone specific logic?

Reply:
Good point. I added a test.